LiskHQ/lisk-mobile
# Update font families for the new design - Closes #777

Please read those two sections of the updated readme first 🤓
- https://github.com/LiskHQ/lisk-mobile/tree/777-update-font-families#development-environment
- https://github.com/LiskHQ/lisk-mobile/tree/777-update-font-families#using-commercial-fonts 

# What was the bug or feature?
Described in #777 

### How did I fix it?
- Updated font constants
- Updated .gitignore and README to contain information about commercial fonts

## Type of change
- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix

### How to test it?
- Make sure you add the new fonts to `src/assets/fonts` folder with the correct naming stated in [`styleGuide/fonts.js`](https://github.com/LiskHQ/lisk-mobile/blob/777-update-font-families/src/constants/styleGuide/fonts.js) file.
- Add `useCommercialFonts: true` to your env.json
- RUN THE APP 🚗 💨 

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
